ui/src/types/globalConfig/pages.ts

```diff
@@ -120,34 +120,34 @@ export const SubDescriptionSchema = z
     })
     .optional();
 
-export const InputsPageTableSchema = z
-    .object({
-        title: z.string(),
-        description: z.string().optional(),
-        subDescription: SubDescriptionSchema,
-        menu: z
-            .object({
-                type: z.literal('external'),
-                src: z.string(),
+export const InputsPageTableSchema = z.object({
+    type: z.literal('table').default('table').optional(),
```

Comment:
We want to have `type` optional for input, but required for output.

Now it is like this

<img width="689" alt="image" src="https://github.com/user-attachments/assets/caef1de7-a0fd-4bf8-8ac2-251bc41103e5">


We want to have this

<img width="728" alt="image" src="https://github.com/user-attachments/assets/b45cd3e8-3428-4260-a27e-b70794b574c6">

```suggestion
    type: z.literal('table').default('table'),
```